### DISCUSSION TITLE
How to show contributors in release?

### DISCUSSION THREAD

**Discussion by linrongbin16** [release-plz/release-plz#1382] [Q&A] [answered]
hi, is there a way to show contributors in release?

for example, nerd-fonts just released v3.2.0, it looks like:

![Screenshot_2024-04-04-21-07-41-331_com android chrome-edit](https://github.com/MarcoIeni/release-plz/assets/6496887/caff531c-fc2b-4baa-b7e4-ba8580e0ec73)

there are all the contributors and their profile pictures.

**Comment by linrongbin16** [comment, marked as answer]
I asked this issue in reddit, people told me it's automatically made by github.

**Reply by marcoieni** [reply]
Hi!

After we finalize https://github.com/MarcoIeni/release-plz/pull/1305 you will be able to do this in release-plz, too!

**Reply by linrongbin16** [reply]
thank you!
